projects/js-packages/shared-extension-utils/src/get-icon-color.js

```diff
@@ -0,0 +1,25 @@
+import colorStudio from '@automattic/color-studio';
```

Comment:
Duplication of `/projects/plugins/jetpack/extensions/shared/block-icons.js`. I'll remove that file once all blocks are using `getBlockIconComponent`.